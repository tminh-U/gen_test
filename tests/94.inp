6
5 18 12 11 9 12 
14
2 6
1 1
2 2
1 1
4 5
6 6
4 4
6 6
4 5
5 6
6 6
2 6
3 5
6 6
