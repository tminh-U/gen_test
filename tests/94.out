9
2
2
2
3
2
2
2
3
4
2
9
4
2
